docs/layers/scenegraph-layer.md

```diff
@@ -0,0 +1,160 @@
+
+<p class="badges">
+  <img src="https://img.shields.io/badge/@deck.gl/mesh--layers-lightgrey.svg?style=flat-square" alt="@deck.gl/mesh-layers" />
+  <img src="https://img.shields.io/badge/lighting-TBD-blue.svg?style=flat-square" alt="lighting" />
+</p>
+
+# ScenegraphLayer
+
+The `ScenegraphLayer` renders a number of instances of a complete luma.gl scenegraph. While scenegraphs can be created programmatically, they are typically obtained by loading glTF files.
+
+```js
+import DeckGL from '@deck.gl/react';
+import {ScenegraphLayer} from '@deck.gl/mesh-layers';
+import {registerLoaders} from '@loaders.gl/core';
+import {GLBScenegraphLoader, GLTFScenegraphLoader} from '@luma.gl/addons';
+
+// Register the proper loader for scenegraph.gltf
+registerLoaders([GLBScenegraphLoader, GLTFScenegraphLoader]);
+
+const App = ({data, viewport}) => {
+
+  /**
+   * Data format:
+   * [
+   *   {
+   *     position: [-122.45, 37.7],
+   *     color: [255, 0, 0]
+   *   },
+   *   {
+   *     position: [-122.46, 37.73],
+   *     color: [0, 255, 0]
+   *   },
+   *   ...
+   * ]
+   */
+  const layer = new ScenegraphLayer({
+    id: 'scenegraph-layer',
+    data,
+    scenegraph: 'scenegraph.gltf'
+  });
+
+  return (<DeckGL {...viewport} layers={[layer]} />);
+};
+```
+
+## Installation
+
+To install the dependencies from NPM:
+
+```bash
+npm install @deck.gl/mesh-layers
+```
+
+```js
+import {ScenegraphLayer} from '@deck.gl/mesh-layers';
+new ScenegraphLayer({});
+```
+
+To use pre-bundled scripts:
+
+```html
+<script src="https://unpkg.com/@deck.gl@~7.0.0/dist.min.js"></script>
+<!-- or -->
+<script src="https://unpkg.com/@deck.gl/core@~7.0.0/dist.min.js"></script>
```

Comment:
@ibgreen My understanding is that this is not going to work unless we release pre-bundled loaders.gl? Shall we remove these for now?

Reply:
Cutting this temporarily sounds good. I think we can add this text back in a deck.gl patch release once we have published scripts for loaders.gl.

Reply:
done